src/runtimes/docker/clients/DockerClientBase/DockerClientBase.ts

```diff
@@ -651,6 +654,42 @@ export abstract class DockerClientBase extends ConfigurableClient implements ICo
 
     //#endregion
 
+    //#region ImagesGenerateSbom Command
+
+    /**
+     * Generate the command line arguments for invoking a pull image command on
+     * a Docker-like client
+     * @param options Pull image command options
+     * @returns Command line arguments for pulling a container image
+     */
+    protected getGenerateSbomCommandArgs(options: ImageGenerateSbomCommandOptions): CommandLineArgs {
+        return composeArgs(

```

Comment:
We should consider whether we want to support the --exclude, --platform, and/or --version arguments.

Reply:
Additionally, there's a --quiet argument that we should look into